netsim/devices/linux.yml

```diff
@@ -15,6 +15,8 @@ features:
     passive: False
   routing:
     static: true
+  initial:
+    ipv6_use_ra: true
```

Comment:
Also need to set this for FRR (at least); used as host in scenarios where VLANs are involved

Reply:
Need to check how those devices behave first.